script/vsts/platforms/templates/get-release-version.yml

```diff
@@ -0,0 +1,19 @@
+parameters:
+  - name: NightlyFlag
+    type: string
+    values:
+      - ' '
+      - --nightly
+    default: ' '
+
+jobs:
+
+- job: GetReleaseVersion
+  pool:
+    vmImage: 'ubuntu-latest'
+  steps:
+    - script: |
+        cd script/vsts
+        npm ci
```

Comment:
I noticed that this is `npm ci` which reverts your change to npm install change. @sadick254.

Reply:
Very briefly: I'd advocate for using `npm ci`.

---

@sadick254 If I recall correctly, the `npm ci` --> `npm install` change was meant to fix intermittent Linux build failures.

It turned out `npm install` doesn't fix those Linux failures. Node 12.16.3 in CI seems to be doing okay for fixing the Linux build failures. (So there isn't a true purpose left as for why to prefer `npm install` over `npm ci`, as I understand it.)

In my personal view, it is slightly advantageous to use `npm ci` iwherever we can. Just a bit faster and cleaner than `npm install`. I admit, though, the speedup is only a few seconds most of the time.

Reply:
But don't worry about that if it slows down merging this. The stakes are so low as to not really matter. To be quite honest.